nano/node/active_elections.cpp

```diff
@@ -416,16 +416,25 @@ nano::election_insertion_result nano::active_elections::insert (std::shared_ptr<
 			debug_assert (count_by_behavior[result.election->behavior ()] >= 0);
 			count_by_behavior[result.election->behavior ()]++;
 
+			// Skip passive phase for blocks without cached votes to avoid bootstrap delays
+			bool active_immediately = false;
+			if (node.vote_cache.find (hash).empty ())
```

Comment:
This should use vote_cache.contains(hash) to avoid an unnecessary allocation

Reply:
Good catch!